### PR TITLE
Update intern guide pages.

### PR DESCRIPTION
**Pull request by Dhruvi16**
## Description

This updates pages under `/internships` section.
This updates - 
1. /internships/applicant-guide/
2. /internships/intern-guide/
3. /internships/mentor-guide/
4. /internships/history/

## Checklist
 
- [x] My pull request has a descriptive title (not a vague title like `Update
  index.md`).
- [x] My pull request targets the `master` branch of the repository.
- [x] My commit messages follow [best practices][best_practices].
- [x] My code follows the established code style of the repository.
- [ ] I added tests for the changes I made (if applicable).
- [ ] I added or updated documentation (if applicable).
- [x] I tried running the project locally and verified that there are no
  visible errors.

[best_practices]:https://gist.github.com/robertpainsi/b632364184e70900af4ab688decf6f53

## Developer Certificate of Origin

<details>
<summary>Developer Certificate of Origin</summary>

```
Developer Certificate of Origin
Version 1.1

Copyright (C) 2004, 2006 The Linux Foundation and its contributors.
1 Letterman Drive
Suite D4700
San Francisco, CA, 94129

Everyone is permitted to copy and distribute verbatim copies of this
license document, but changing it is not allowed.


Developer's Certificate of Origin 1.1

By making a contribution to this project, I certify that:

(a) The contribution was created in whole or in part by me and I
    have the right to submit it under the open source license
    indicated in the file; or

(b) The contribution is based upon previous work that, to the best
    of my knowledge, is covered under an appropriate open source
    license and I have the right under that license to submit that
    work with modifications, whether created in whole or in part
    by me, under the same open source license (unless I am
    permitted to submit under a different license), as indicated
    in the file; or

(c) The contribution was provided directly to me by some other
    person who certified (a), (b) or (c) and I have not modified
    it.

(d) I understand and agree that this project and the contribution
    are public and that a record of the contribution (including all
    personal information I submit with it, including my sign-off) is
    maintained indefinitely and may be redistributed consistent with
    this project or the open source license(s) involved.
```

</details>

### CODE REVIEW

**Comment by Dhruvi16** [comment]
I am not sure about what logo can we use. @panchovm Can you suggest anything here?

File: content/internships/contents.lr [hunk 1]
```diff
@@ -43,15 +43,64 @@ If you're interested in this program, please review the resources below and come
...
-| Brent Moran 	                | `@Brent`                |	[See CC Open Source Blog author page](/blog/authors/mathemancer/)                |
-| Dhruv Bhanushali 	            | `@dhruvkb` 	            | [See CC Open Source Blog author page](/blog/authors/dhruvkb/)                    |
-| Hugo Solar 	                  | `@hugo`                 |	[See CC Open Source Blog author page](/blog/authors/hugosolar/)                  |
-| Kriti Godey 	                | `@kriti` 	              | [See CC Open Source Blog author page](/blog/authors/kgodey/)                     |
-| María Belén Guaranda Cabezas 	| `@Maria Belen Guaranda` |	[See CC Open Source Blog author page](/blog/authors/soccerdroid/)                |
-| Sarah Hinchcliff Pearson 	    | `@shinchpearson`        |	[See CC staff bio](https://creativecommons.org/author/sarahcreativecommons-org/) |
-| Timid Robot Zehta 	          | `@Timid Robot`          |	[See CC Open Source Blog author page](/blog/authors/TimidRobot/)                 |
+<div class="table-container padding-vertical-normal">
```
Comment (hugosolar) [inline content/internships/contents.lr:46]: It didn't work if you surround the markdown format with the `.table-container` class?

**Comment by panchovm** [comment]
There is a long-term solution that implies working on a responsive version. We can transform the table after a specific breakpoint where each row changes as a mini-table. Something [like this](https://codepen.io/team/css-tricks/pen/wXgJww?editors=1100).

I like this idea, and it matches the conversation we had regarding responsive components.

**Comment by hugosolar** [comment]
> There is a long-term solution that implies working on a responsive version. We can transform the table after a specific breakpoint where each row changes as a mini-table. Something [like this](https://codepen.io/team/css-tricks/pen/wXgJww?editors=1100).
> 
> I like this idea, and it matches the conversation we had regarding responsive components.

@Dhruvi16 I think this is a great Idea actually.. what do you think?

**Comment by Dhruvi16** [comment]
> > There is a long-term solution that implies working on a responsive version. We can transform the table after a specific breakpoint where each row changes as a mini-table. Something [like this](https://codepen.io/team/css-tricks/pen/wXgJww?editors=1100).
> > I like this idea, and it matches the conversation we had regarding responsive components.
> 
> @Dhruvi16 I think this is a great Idea actually.. what do you think?

Indeed it is a great idea.

**Comment by Dhruvi16** [comment]
But shouldn't it be implemented in Vocabulary itself? @panchovm

**Comment by hugosolar** [comment]
> But shouldn't it be implemented in Vocabulary itself? @panchovm @hugosolar

Yes, you're right, it should be implemented in Vocabulary, I'll approve this PR then, and you can open the issue onm Vocabulary repo

**Pull request merged**